File: tabornik-frontend/src/app/model/camp-worker.ts
export class CampWorker {
    campId: number;
    workerId: number;
    position: String;
    pending: Boolean;

    constructor(campId: number, workerId: number, position: String, pending: Boolean) {
        this.campId = campId;
        this.workerId = workerId;
        this.position = position;
        this.pending = pending;
    }
}

File: tabornik-backend/app/models/CampWorker.scala
package models

import play.api.libs.json.Json

object CampWorker {
  implicit val format = Json.format[CampWorker]
}

case class CampWorker(campId: Int, workerId: Int, pending: Boolean, position: String)


object CampWorkerUnit {
  implicit val format = Json.format[CampWorkerUnit]
}
case class CampWorkerUnit(worker: Worker, position: String)

